src/data/dapp_promotions.json

```diff
@@ -1,14 +1,8 @@
 [
   {
-    "name": "Algem - Liquid Staking",
-    "shortDescription": "Receive one liquid nASTR token for every ASTR token you stake. Then use nASTR to yield farm on the ecosystem to maximize your earnings.",
-    "link": "https://www.algem.io/",
-    "img": "images/dapp_promotions/algem.jpg"
-  },
-  {
-    "name": "ASTR token, value and utility",
-    "shortDescription": "How the ASTR Token Brings Utility and Value to the Astar Ecosystem",
-    "link": "https://astar.network/blog/how-the-astr-token-brings-utility-and-value-to-the-astar-39387",
-    "img": "https://ipfs.subsocial.network/ipfs/bafybeig2o6zjlnxnwgdv6xkcgu4w5tn7wdre26kimpc2w2ldrum66cotw4"
+    "name": "Age Of Chronos",
```

Comment:
Don't delete other promotions. Just add yours to the array.

Since the promotion image is on IPFS, you can delete it from the portal codebase

Reply:
I have made a new commit based on your feedback. Thank you